test/TestCase.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Intercom\Test;
+
+use Intercom\IntercomClient;
+use PHPUnit\Framework\TestCase as BaseTestCase;
+
+abstract class TestCase extends BaseTestCase
+{
+    /**
+     * @var \Intercom\IntercomClient|\PHPUnit\Framework\MockObject\MockObject
```

Comment:
Tiny thing, you're importing `IntercomClient`, so we don't need the full path here

```suggestion
     * @var IntercomClient|\PHPUnit\Framework\MockObject\MockObject
```